### PR TITLE
26 - [feat] Parse NYISO link from HTML

### PR DESCRIPTION
**Pull request by deenasun**
### Description
[//]: # "[Required] Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

Updated `api/webscraper/nyiso_scraper.py`
- Use BeautifulSoup library to parse HTML of https://www.nyiso.com/connecting-to-the-grid to get the link to download the NYISO xlsx spreadsheet

### Screenshots
[//]: # "Add any screenshots that will help your PR reviewer understand what’s going on in the PR!"
[//]: # "[Required] For frontend tasks: add screenshots of the layout and expected behavior (GIFs if you’re feeling extra fancy)!"
[//]: # "Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

[image placeholder]: 

## How to review
[//]: # "/[Required] Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?"

```
git fetch origin
git checkout 26-feat-parse-nyiso-link-from-html
```
In the root directory, run this command to download all the necessary python dependencies:
```
pip install -r requirements.txt
```

## Next steps
[//]: # "[Optional] - What has not yet been added to the PR, doesn’t yet have any functionality — essentially, what needs to be fixed or added."
[//]: # "[Optional] any temporary fixes in this PR that should be cleaned up later."

### Online sources
[//]: # "[Optional] insert links to any useful tutorials or documentation that you used for this PR!"

### Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @itsliterallymonique

### CODE REVIEW

File: api/webscraper/database_constants.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+renewable_energy_set = {
+    "Hydroelectric",
+    "Land Based Wind",
```
Comment (itsliterallymonique) [inline api/webscraper/database_constants.py:3]: let's make this Land-Based Wind

File: api/webscraper/database_constants.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    "Energy Storage",
+    "Pumped Storage",
+}
+
+renewable_energy_map = {
+    "H": "Hydroelectric",
+    "S": "Solar",
+    "ES": "Energy Storage",
+    "PS": "Pumped Storage",
+    "W": "Land Based Wind",
```
Comment (itsliterallymonique) [inline api/webscraper/database_constants.py:16]: Land-Based Wind

**Review by itsliterallymonique — CHANGES_REQUESTED** [review]
Just make 2 more quick changes from Land Based Wind to Land-Based Wind

File: api/webscraper/database_constants.py (hunk 1 above)
Comment (deenasun) [reply api/webscraper/database_constants.py]: Done!

**Review by itsliterallymonique — APPROVED** [review]
amazing job Deena. you can squash and merge when you are ready!

**Pull request merged**